src/MySql.DataLayer.Core/Repository/BaseMySqlRepository.cs

```diff
@@ -427,5 +430,48 @@ public virtual async Task<int> UpdateAsync(TEntity entity, MySqlConnection conne
 
             return await connection.ExecuteAsync(_sql.ToString(), dapperParameters, transaction);
         }
+        /// <summary>
+        /// Provides the execution of a StoredProcedure to get a list for mapped data
+        /// </summary>
+        /// <typeparam name="TResult"></typeparam>
+        /// <param name="queryParameters"></param>
+        /// <returns></returns>
+        public virtual async Task<List<TResult>> ExecuteStoredProcedure<TResult>(QueryParameter[] queryParameters = null)
```

Comment:
Work with Dapper `DynamicParameters` with parameter and value to pass in procedure.
Add the `CommandType.StoredProcedure` in `QueryAsync`